docs/source/user/fast.farm/FFarmTheory.rst

```diff
@@ -1291,6 +1292,135 @@ the solution of the momentum equation),
+
+.. math::
+
+   \begin{aligned}
+      k_{} (x,y,z) = 
+          \frac{k_\text{def} }{\overline{U}} \left| V_x^{Wake}(x,y,z) \right|
+        + \frac{k_\text{grad}D}{2\overline{U}} \left[\left|{\frac{\partial {V_x^{Wake}(x,y,z)}}{\partial r}}\right| +  \left|{\frac{1}{r}\frac{\partial {V_x^{Wake}(x,y,z)}}{\partial \theta}}\right|  \right]  
+        \end{aligned}
+
+where :math:`D` is a reference diameter, and :math:`\bar{U}` is the mean velocity taken as the filtered velocity at the turbine location normal to the rotor disk.
+The coordinates :math:`x,y,z` and :math:`r,\theta` are taken in the meandering frame of reference.
+The parameters :math:`k_\text{def}` and :math:`k_\text{grad}` are tuning constant of the model. They will be calibrated in future studies.
+
+
+
+**Unit turbulence boxes** 
+
+The 3 turbulence Mann boxes are stored as a 4D
+array :math:`\boldsymbol{u}_\text{unit}` of dimension
+:math:`(3,n_x, n_y, n_z)`.
+The turbulence boxes used for the WAT are isotropic turbulence boxes
+with unit standard deviation, generated using the Mann
+model :cite:`ff-Mann1994`.
+To generate a box with unit standard deviation, the dissipation rate is set to:
+
+.. math::
+
+      \alpha\epsilon^{2/3}\approx \frac{1}{0.688 L^{2/3}} 
+
+We have found that there is no dependency on the length scale. We nevertheless recommend to set it to the rotor diameter if the users generate their own boxes.
+
+
+**Predefined boxes**
+
+A recommended practice for the high-resolution domain of FAST.Farm is to chose a grid
+spacing equal to the maximum chord of the blade. 
+Based on the data from
+different wind turbine, the max-chord can be approximated as:
+:math:`c_\text{max}\approx 0.03D`. 
+Therefore we suggest to use this spacing in all three directions, and as a compromise to obtain a box
+with sufficient extent but moderate size, we select: :math:`\Delta x = \Delta y = \Delta z = 0.03D`,
+:math:`L_x = L_y=15D`, :math:`L_z= 2D`, :math:`n_x=n_y=512`,
+:math:`n_z=64`, leading to a box size of :math:`65` Mb per wind
+component.
+
+Users may generate their own Mann box using the guidelines presented in this paragraph and the one above. 
+
+
+
+**Convection of the WAT box**
+
+There is only one WAT turbulence box stored for the entire wind farm. To
+convect the WAT turbulence box, the AWAE module keeps track of a passive
+tracer that is convected at each time step with the mean of the
+rotor-average velocity of each wind turbine
+:math:`(\boldsymbol{U}_\text{farm}`). The position of the passive
+tracer, :math:`\boldsymbol{B}`, is defined as:
+
+.. math::
+
+      \frac{d\boldsymbol{B}}{dt} = \boldsymbol{U}_\text{farm}(t)
+
+where:
+
+.. math::
+
+      \boldsymbol{U}_\text{farm} = \operatorname{mean}\{ \overline{V}^\text{Low}_\text{Amb}[i_w], i_w =1\cdots n_{WT}\}
+
+where :math:`\overline{V}^\text{Low}_\text{Amb}[i_w]` is the rotor
+averaged ambient wind speed. 
```

Comment:
Suggest noting here that the convection of the turbulence box may change if you add or remove turbines from the wind farm.